### ISSUE TITLE
Update an article which has an empty body

### ISSUE THREAD

**Issue by maxime1992** [maxime1992/dev-to-git#19]
If an article has an empty body, it should be able to update (which is not the case)

```
Error name: "Error"
Error message: ""
Error stack: "Error
    at Article.<anonymous> (/home/maxime/Documents/perso/projects/my-dev.to/node_modules/dev-to-git/bin/dev-to-git.umd.js:11180:39)
    at step (/home/maxime/Documents/perso/projects/my-dev.to/node_modules/dev-to-git/bin/dev-to-git.umd.js:77:27)
    at Object.next (/home/maxime/Documents/perso/projects/my-dev.to/node_modules/dev-to-git/bin/dev-to-git.umd.js:58:57)
    at fulfilled (/home/maxime/Documents/perso/projects/my-dev.to/node_modules/dev-to-git/bin/dev-to-git.umd.js:48:62)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)"
```

To repro:

- create a draft on dev.to
- set a title but no body
- try to update that article from the command line with `yarn run dev-to-git --dev-to-token YOUR-TOKEN`

**Comment by gabbersepp** [comment]
I encountered the same situation the last few times I write a new blog post. First it took some time to find out what was wrong :D

**Comment by maxime1992** [comment]
Should be fixed by https://github.com/maxime1992/dev-to-git/pull/49 (thanks @xhiroga :clap:) _(and a [tiny follow up](https://github.com/maxime1992/dev-to-git/pull/50))_.

The fix should be available in [v1.4.9](https://github.com/maxime1992/dev-to-git/releases/tag/v1.4.9)
